src/ecl/eclWatchTree.ts

```diff
@@ -182,6 +186,11 @@ export class ECLWatchTree extends Tree {
         vscode.commands.executeCommand("setContext", "hpccPlatform.isAllWorkunits", !this._myWorkunits);
     }
 
+    userRefresh(): void {
+        this.updateMenu();
+        super.refresh();
+    }
```

Comment:
This method isn't needed, you can just call `refresh()`?

Reply:
Actually, that would work!

Reply:
The whole `userRefresh` method is not needed

Reply:
Yes it is. If you call it from package.json, the refresh that is attached to the wu treeview item will send the item and it will check if the item has changed, and will not force the refresh.